addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/TemperatureModel.java

```diff
@@ -30,24 +31,30 @@
     private BigDecimal offset;
 
     public BigDecimal getCelsius() {
-        return celsius != null ? celsius.multiply(TEMP_FACTOR) : BigDecimal.ZERO;
```

Comment:
Yes you are right. And while I am thinking about it - and looking into the documentation of the AHA HTTP Interface again - I came to the conclusion that I will revert this change. The FRITZ!Box returns already celsius values but moved by one decimal place. The values only have to be multiplied by the TEMP_FACTOR.